inductiva/simulation/simulator.py

```diff
@@ -88,7 +88,7 @@ def run_async(
             **kwargs: Additional keyword arguments to be passed to the
                 simulation API method.
         """
-        input_dir, _ = self._setup_dirs(input_dir, None)
+        input_dir, _ = self._setup_io_dirs(input_dir, None)
```

Comment:
In principle this is fine, but notice that you are performing the operation of creating an output_dir name with this.

Reply:
Yes, but I wanted to avoid creating an additional function because for the setup of the output_dir name also needs the input_dir name.